docs/contributors/maintainers.md

```diff
@@ -30,6 +30,8 @@ are automated once the release is triggered but manual actions are required for:
   - [release-drafter](https://github.com/release-drafter/release-drafter) creates or updates draft release for the
   next version each time a pull request is merged to the `master` branch.
   - Rename the existing draft release to `In Progress`. The name is not relevant and will be later used to identify the draft release to update.
+- Create a new draft release and name it `Notes for next release notes`. It will be used during development to note important things and prepare
```

Comment:
Following the action to do at each release, we should have already a draft release note named `Notes for next release notes` for the created at the last release.

I think we should create it after the current release notes is published.

Reply:
No we cannot. Otherwise, it will become the youngest one and release-drafter will update it. It must be older than the one that is constantly updated by release-drafter.

Reply:
Ok, but In this step, we have 2 release notes with the name `Notes for next release notes`.

So, fot the next step where we should remove the old, it might be confusing 🙁

Reply:
I agree. I am going to propose a simplified way of using such release notes.

Reply:
b1af1ff7af00b377263d2f20678d65ed6b755e81 should improve the explanations. Please tell me if it is clearer now @csouchet